### PR TITLE
Added some titles, did some small changes

### PR DESCRIPTION
**Pull request by Metroid55**
Don't know much about lua, hope I didn't break anything.
Also the snowball and snap titels aren't working rn, but idk how to fix that.

### CODE REVIEW

File: lua/swamp/misc/sh_titles.lua [hunk 1]
```diff
@@ -300,30 +302,31 @@ AddTitle({
...
 
 AddTitle({
-    {100, "Shanker", 0},
-    {1000, "Edge Lord", 0},
-    {5000, "Throat-Neck Slitter", 0},
-    {10000, "American Psycho", 0}
+    {100, "Shanker"},
+    {1000, "Edge Lord"},
+    {5000, "Throat-Neck Slitter"},
+    {10000, "American Psycho", 2000}
```
Comment (Noztik) [inline lua/swamp/misc/sh_titles.lua:323]: what's with the reward

Comment (Metroid55) [reply lua/swamp/misc/sh_titles.lua:323]: get a free knife after 10k knife kills as a troll

**Comment by swampservers** [comment]
Needs update

**Comment by Metroid55** [comment]
Okay that should be 6 new titles and some minor adjustments, if you want something changed just tell me

**Comment by Metroid55** [comment]
I'm going to close this for now and make a new pull request in a few days when I'm back home

**Pull request closed without merging**